lava-docker-master-image/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM bitnami/minideb:stretch
+
+RUN echo "deb http://ftp.fr.debian.org/debian/ stretch-backports main" >> /etc/apt/sources.list
+
+RUN apt-get update
+
+# Install debian packages used by the container
+# Configure apache to run the lava server
+# Log the hostname used during install for the slave name
+RUN echo 'lava-server   lava-server/instance-name string lava-docker-instance' | debconf-set-selections \
+ && echo 'locales locales/locales_to_be_generated multiselect C.UTF-8 UTF-8, en_US.UTF-8 UTF-8 ' | debconf-set-selections \
+ && echo 'locales locales/default_environment_locale select en_US.UTF-8' | debconf-set-selections
+
+# e2fsprogs is for libguestfs
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+ locales \
+ postgresql \
+ screen \
+ sudo \
+ wget \
+ e2fsprogs \
+ gnupg \
+ vim
+
+RUN wget http://images.validation.linaro.org/production-repo/production-repo.key.asc \
+ && apt-key add production-repo.key.asc \
+ && echo 'deb http://images.validation.linaro.org/production-repo/ stretch-backports main' > /etc/apt/sources.list.d/lava.list \
+ && apt-get clean && apt-get update
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+ qemu-system \
+ qemu-system-arm \
+ qemu-system-i386 \
+ ser2net \
+ u-boot-tools \
```

Comment:
I don't think ser2net or u-boot-tools are needed on the master.

Reply:
For that matter, I don't think the QEMU stuff is needed either when the master is a "dumb" master (no devices.)

Reply:
removed from master